openapi/components/schemas/auth/RegisterOrgRequest.yml

```diff
@@ -0,0 +1,14 @@
+type: "object"
+required:
+  - organizationName
+properties:
+  organizationName:
```

Comment:
Is it decided to use `camelCase` for query params, fields and payloads?

Reply:
yes vai. we decided to use camelCase